skfair/metrics/fairness_report.py

```diff
@@ -0,0 +1,117 @@
+from functools import partial
+from collections import defaultdict
+
+from sklearn.metrics import (confusion_matrix, f1_score, precision_score,
+                             accuracy_score, recall_score)
+import numpy as np
+import pandas as pd
+
+
+def false_positive_score(y_true, y_pred, labels=None):
+    conf_matrix = confusion_matrix(y_true, y_pred, labels=labels)
+    tn, fp, fn, tp = true_false_positive_negative(conf_matrix)
+    eps = 1e-10
+    return fp / (fp + tn + eps)
+
+
+def false_discovery_score(y_true, y_pred, labels=None):
+    conf_matrix = confusion_matrix(y_true, y_pred, labels=labels)
+    tn, fp, fn, tp = true_false_positive_negative(conf_matrix)
+    eps = 1e-10
+    return fp / (tp + fp + eps)
+
+
+DEFAULT_METRICS = {
+    "TPR": partial(recall_score, average="micro"),
+    "FPR": false_positive_score,
+    "PPVR": partial(precision_score, average="micro"),
+    "FDR": false_discovery_score,
+    "ACC": accuracy_score,
+    "F1": partial(f1_score, average="micro")
+}
+
+
+def true_false_positive_negative(conf_matrix):
+    # conf matrix: rows are true, cols are pred
+    if conf_matrix.shape == (2, 2):  # binary case
+        tn, fp, fn, tp = conf_matrix.ravel()
+        return tn, fp, fn, tp
+    tp = 0
+    fp = 0
+    fn = 0
+    tn = 0
```

Comment:
Do we *need* this code here? it seems that the code below handles the general case and I cannot imagine that there's such a substantial performance boost.

Reply:
You mean removing these lines right?
```
if conf_matrix.shape == (2, 2):  # binary case
        tn, fp, fn, tp = conf_matrix.ravel()
        return tn, fp, fn, tp
```

Reply:
Ah sorry, I selected too many lines there. But yes :)